pkg/controller/registry/reconciler/grpc.go

```diff
@@ -17,13 +20,25 @@ import (
 	"k8s.io/apimachinery/pkg/util/intstr"
 )
 
-const CatalogSourceUpdateKey = "catalogsource.operators.coreos.com/update"
+const (
+	CatalogSourceUpdateKey      = "catalogsource.operators.coreos.com/update"
+	CatalogPollingRequeuePeriod = 30 * time.Second
```

Comment:
this seems on the long side if waiting for a catalog pod to start - I would think 1-5s would be more typical

Reply:
I had set it originally to 10 seconds but then we upped it because as @dinhxuanvu pointed out it takes time for the new image to be pulled down onto the node. Of course several layers would already be cached and shared between the new/old image but I guess we wanted to stay on the safe side here.